microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -44,9 +44,9 @@ The DevTools extension for Visual Studio Code is one of several different ways t
 
 | Approach | Pros and Cons | Article |
 |---|---|---|
-| Browser > DevTools > **Elements** tool | You have to manually copy changes from DevTools into your source files. | [Inspect, edit, and debug HTML and CSS with the Elements tool](/microsoft-edge/devtools-guide-chromium/elements-tool/elements-tool.md) |
-| Browser > DevTools > **Sources** tool > **Filesystem** tab to define a **Workspace** | Always saves the changes, which could discourage experimenting | [Using the Filesystem tab to define a local Workspace](/microsoft-edge/devtools-guide-chromium/sources/index.md#using-the-filesystem-tab-to-define-a-local-workspace) in _Sources tool overview_ |
-| Browser > DevTools > **Settings** > **Experiments** > **Open source files in Visual Studio Code** | Always saves the changes, which could discourage experimenting | Experiments > [Opening source files in Visual Studio Code](/microsoft-edge/devtools-guide-chromium/sources/opening-sources-in-vscode.md). |
+| Browser > DevTools > **Elements** tool | You have to manually copy changes from DevTools into your source files. | [Inspect, edit, and debug HTML and CSS with the Elements tool](/microsoft-edge/devtools-guide-chromium/elements-tool/elements-tool) |
```

Comment:
Global for PR: To link to another page in this repo, use "Relative" path pattern (dots on left, .md on right) like:
../devtools-guide-chromium/elements-tool/elements-tool.md
Test the link in VS Code by Ctrl+Click.  Supports autocompletion.  
Or use Alt+M to add a link: **Link > Link to file in repo**.

Reply:
fixed in the pr